### PR TITLE
Updated Moesif plugin docs and logo. 

### PR DESCRIPTION
**Pull request by dgilling**

### CODE REVIEW

**Comment by tlblessing** [comment]
Direct review link:  https://deploy-preview-2079--kongdocs.netlify.app/hub/moesif/kong-plugin-moesif/

File: app/_hub/moesif/kong-plugin-moesif/index.md [hunk 1]
```diff
@@ -63,39 +59,59 @@ params:
       required: true
       default:
       value_in_examples: MY_MOESIF_APPLICATION_ID
-      description: The Moesif application token provided to you by [Moesif](http://www.moesif.com).
+      description: Your Moesif Application Id from your [Moesif](http://www.moesif.com) dashboard. Go to Top Right Menu -> Installation.
     - name: api_endpoint
       required: false
       default: "`https://api.moesif.net`"
-      description: URL for the Moesif API.
+      description: URL for the Moesif Collection API (Only modify if secure proxy is used)
```
Comment (tlblessing) [inline app/_hub/moesif/kong-plugin-moesif/index.md:66]:
```suggestion
      description: URL for the Moesif Collection API (Only modify if secure proxy is used).
```

File: app/_hub/moesif/kong-plugin-moesif/index.md [hunk 2]
```diff
@@ -63,39 +59,59 @@ params:
...
     - name: timeout
       required: false
       default: "`10000`"
-      description: An optional timeout in milliseconds when sending data to Moesif.
+      description: An optional timeout in milliseconds when sending data to Moesif
```
Comment (tlblessing) [inline app/_hub/moesif/kong-plugin-moesif/index.md:70]:
```suggestion
      description: An optional timeout in milliseconds when sending data to Moesif.
```

Should be consistent punctuation throughout table.

File: app/_hub/moesif/kong-plugin-moesif/index.md [hunk 3]
```diff
@@ -63,39 +59,59 @@ params:
...
     - name: keepalive
       required: false
-      default: "`5000`"
-      description: An optional value in milliseconds that defines for how long an idle connection will live before being closed.
+      default: "`10000`"
+      description: An optional value in milliseconds that defines for how long an idle connection will live before being closed
```
Comment (tlblessing) [inline app/_hub/moesif/kong-plugin-moesif/index.md:74]:
```suggestion
      description: An optional value in milliseconds that defines for how long an idle connection will live before being closed.
```

File: app/_hub/moesif/kong-plugin-moesif/index.md [hunk 4]
```diff
@@ -63,39 +59,59 @@ params:
...
     - name: api_version
       required: false
       default: "`1.0`"
       description: An optional API Version you want to tag this request with
     - name: disable_capture_request_body
       required: false
       default: "`false`"
-      description: An option to disable logging of request body
+      description: Set to true to disable logging of request body
```
Comment (tlblessing) [inline app/_hub/moesif/kong-plugin-moesif/index.md:82]:
```suggestion
      description: Set to true to disable logging of request body.
```

File: app/_hub/moesif/kong-plugin-moesif/index.md [hunk 5]
```diff
@@ -63,39 +59,59 @@ params:
...
     - name: disable_capture_response_body
       required: false
       default: "`false`"
-      description: An option to disable logging of response body
-    - name: request_masks
+      description: Set to true to disable logging of response body
```
Comment (tlblessing) [inline app/_hub/moesif/kong-plugin-moesif/index.md:86]:
```suggestion
      description: Set to true to disable logging of response body.
```

File: app/_hub/moesif/kong-plugin-moesif/index.md [hunk 6]
```diff
@@ -63,39 +59,59 @@ params:
...
+    - name: request_header_masks
+      required: false
+      default: "`{}`"
+      description: An array of request header fields to mask
```
Comment (tlblessing) [inline app/_hub/moesif/kong-plugin-moesif/index.md:90]:
```suggestion
      description: An array of request header fields to mask.
```

File: app/_hub/moesif/kong-plugin-moesif/index.md [hunk 7]
```diff
@@ -63,39 +59,59 @@ params:
...
+    - name: request_header_masks
+      required: false
+      default: "`{}`"
+      description: An array of request body fields to mask
```
Comment (tlblessing) [inline app/_hub/moesif/kong-plugin-moesif/index.md:94]:
```suggestion
      description: An array of request body fields to mask.
```

File: app/_hub/moesif/kong-plugin-moesif/index.md [hunk 8]
```diff
@@ -63,39 +59,59 @@ params:
...
+    - name: response_header_masks
       required: false
       default: "`{}`"
-      description: An option to mask a specific request body field
-    - name: response_masks
+      description: An array of response header fields to mask
```
Comment (tlblessing) [inline app/_hub/moesif/kong-plugin-moesif/index.md:98]:
```suggestion
      description: An array of response header fields to mask.
```

File: app/_hub/moesif/kong-plugin-moesif/index.md [hunk 9]
```diff
@@ -63,39 +59,59 @@ params:
...
+    - name: response_body_masks
       required: false
       default: "`{}`"
-      description: An option to mask a specific response body field
+      description: An array of response body fields to mask
```
Comment (tlblessing) [inline app/_hub/moesif/kong-plugin-moesif/index.md:102]:
```suggestion
      description: An array of response body fields to mask.
```

File: app/_hub/moesif/kong-plugin-moesif/index.md [hunk 10]
```diff
@@ -63,39 +59,59 @@ params:
...
+    - name: debug
+      required: false
+      default: false
+      description: An option if set to true, prints internal log messages for debugging integration issues.
+    - name: user_id_header
+      required: false
+      default: ""
+      description: An optional field name to identify User from a request or response header.
```
Comment (tlblessing) [inline app/_hub/moesif/kong-plugin-moesif/index.md:110]:
```suggestion
      description: An optional field name to identify a User from a request or response header.
```

File: app/_hub/moesif/kong-plugin-moesif/index.md [hunk 11]
```diff
@@ -63,39 +59,59 @@ params:
...
+    - name: company_id_header
+      required: false
+      default: ""
+      description: An optional field name to identify Company (Account) from a request or response header.
```
Comment (tlblessing) [inline app/_hub/moesif/kong-plugin-moesif/index.md:114]:
```suggestion
      description: An optional field name to identify a Company (Account) from a request or response header.
```

File: app/_hub/moesif/kong-plugin-moesif/index.md [hunk 12]
```diff
@@ -112,14 +128,14 @@ params:
...
+[Moesif API Analytics](https://www.moesif.com/?language=kong-api-gateway). Use Moesif
+to understand API adoption and resolve issues quickly.
 
 [Package on Luarocks](http://luarocks.org/modules/moesif/kong-plugin-moesif)
 
+Moesif natively supports REST, GraphQL, Web3, SOAP, JSON-RPC, and more.
+
 ### How to install
 
 The .rock file is a self contained package that can be installed locally or from a remote server.
```
Comment (tlblessing) [inline app/_hub/moesif/kong-plugin-moesif/index.md:141]:
```suggestion
The `.rock` file is a self-contained package that can be installed locally or from a remote server.
```

**Comment by dgilling** [comment]
Thanks @tlblessing, all the punctuation suggestions look good.

**Comment by dgilling** [comment]
Hi @tlblessing, I pushed up the suggested grammer changes already. Let me know if something else is needed from my end.

**Pull request merged**